pkg/virtctl/pause/pause.go

```diff
@@ -122,7 +121,13 @@ func (vc *VirtCommand) Run(cmd *cobra.Command, args []string) error {
 			}
 			err = virtClient.VirtualMachineInstance(namespace).Pause(vmiName)
 			if err != nil {
-				return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v", vmiName, err)
+				if vm.Spec.Running != nil && !*vm.Spec.Running {
+					return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v. VirtualMachine %s is not set to running", vmiName, err, vmiName)
+				} else if vm.Spec.Running != nil && *vm.Spec.Running {
+					return fmt.Errorf("Error pausing VirutalMachineInstance %s: %v", vmiName, err)
```

Comment:
When the VM is started (and this is within the NotFound check), you can just return that the VirutalMachineInstance was not found (yet?), without err in the message.